### ISSUE TITLE
Tab-navigation through docs' sidebar focuses on some invisible items

### ISSUE THREAD

**Issue by smellyshovel** [epicmaxco/vuestic-ui#829]
* **What is the expected behavior?**
There should be no invisible something to be focused on between "Configuration Guide" and "Styles" items when navigating using the keyboard.

* **What is the current behavior?**
![Peek 2021-06-03 12-33](https://user-images.githubusercontent.com/9782236/120628745-b7db1600-c46d-11eb-9c7c-32fe39d3263f.gif)

* **Steps to reproduce, ideally also a screenshot or gif if it's a style issue**

* **Other information**

**Comment by smellyshovel** [comment]
Continues #810 : the items are now properly highlighted, though the work is not yet completed in general.

**Comment by smellyshovel** [comment]
Solved in #828 

![Peek 2021-06-03 12-33](https://user-images.githubusercontent.com/9782236/120648950-e1536c00-c484-11eb-831d-6adc5f426bf5.gif)

The "invisible element" happened to be the `va-collapse`'s header (i.e. the thing you click to open the collapse).

I've proposed in #828 to add a `focusable` property (`true`) by default which (when is set to `false`) disables keyboard-focusing the header element. That's usefull for the cases like this issue describes when there's another component is used as a header and the focus should be gained by it instead.

Basically, the `focusable` property when set to `false` simply sets the `tabindex` to `-1`.
